src/main/java/foxtrot/uniform/charlie/kilo/model/dto/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package foxtrot.uniform.charlie.kilo.model.dto;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public List<Long> getTicketIds() {
+        return ticketIds;
+    }
+
+    public void setTicketIds(List<Long> ticketIds) {
+        this.ticketIds = ticketIds;
+    }
+
+    @Override
+    public String toString() {
+        return "ShoppingCartResponseDto{" + "id=" + id
+                + ", ticketIds=" + ticketIds
+                + ", userId=" + '}';
```

Comment:
```suggestion
                + '}';
```

Reply:
Sleep is a superpower but this is the result when you are a human among heroes.